docs/traefik/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3.9'
```

Comment:
I wonder if we can put the traefik and network related files into the virtual-machines repo and just point to that repo. Is it here because this is open source? We can probably add alt names for all the projects and make it work for everybody.

Reply:
Yeah, only leaving it in the repo because some users don't have access to the virtual machines repo (which is closed source.)